pyproject.toml

```diff
@@ -76,8 +82,10 @@ skip_glob = ["tests/fixtures/common/collections*"] # Skip ansible content due to
     ]
 
     [tool.pylint.master]
-    # tm_tokenize is virtually vendored and shouldn't be linted as such
-    ignore = "tm_tokenize"
+    ignore = [
+      "/src/ansible_navigator/_version.py", # built by setuptools_scm
```

Comment:
Does it work with a leading slash? I know that it is significant in gitignore but not sure about this config.

Reply:
Ha!  No it doesn't.....I'm running from source without having built/installed navigator.